external-import/comlaude/src/main.py

```diff
@@ -285,16 +305,13 @@ def run(self):
             self.comlaude_search.get_next_page()
             self._iterate_events()
 
-        if self.helper.connect_run_and_terminate:
```

Comment:
Why did you remove the run_and_terminate condition ?

Reply:
Hello Megradredo, i deleted the run and terminate condition because with this the script did a loop all the time